packages/atti-components/src/components/Menu/styles/MenuHolder.js

```diff
@@ -0,0 +1,59 @@
+import styled, { css } from 'styled-components'
+
+import verticalMenu from './verticalMenu'
+
+import { ItemLink } from '../components/Item/styles'
+import { Arrow, ItemsContainer } from '../components/ItemGroup/styles'
+
+const MenuHolder = styled.div`
```

Comment:
You can reduce heavily the code base of this file by centralizing the arrow functions

Reply:
Done